documentation/examples/prometheus.yml

```diff
@@ -27,4 +27,17 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ['localhost:9090']
+      - targets: 
+        - 'localhost:9090'
+
+  # An example for relabeling an instance
```

Comment:
I'd not add anything like this to this file, in fact I want to trim this file down further as it's one of the first configs new users tend to come across.